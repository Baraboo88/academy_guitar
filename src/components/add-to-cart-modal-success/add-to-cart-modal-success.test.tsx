import * as renderer from 'react-test-renderer';
import * as React from 'react';


import AddToCartModalSuccess from './add-to-cart-modal-success';
import {BrowserRouter} from 'react-router-dom';

it('AddToCartModalSuccess successfully rendered', () => {
  const mockCloseHandler = jest.fn();

  const tree = renderer.create(<BrowserRouter><AddToCartModalSuccess onModalClose={mockCloseHandler}/></BrowserRouter>);
  expect(tree).toMatchSnapshot();
});
