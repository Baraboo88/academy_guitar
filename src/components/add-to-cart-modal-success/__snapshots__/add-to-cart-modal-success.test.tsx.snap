// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToCartModalSuccess successfully rendered 1`] = `
<div
  className="modal is-active modal--success modal-for-ui-kit"
>
  <div
    className="modal__wrapper"
  >
    <div
      className="modal__overlay"
      data-close-modal={true}
      data-test="test-close-modal"
      onClick={[MockFunction]}
    />
    <div
      className="modal__content"
    >
      <svg
        aria-hidden="true"
        className="modal__icon"
        height="20"
        width="26"
      >
        <use
          xlinkHref="#icon-success"
        />
      </svg>
      <p
        className="modal__message"
      >
        Товар успешно добавлен в корзину
      </p>
      <div
        className="modal__button-container modal__button-container--add"
      >
        <a
          className="button button--small modal__button"
          href="/cart"
          onClick={[Function]}
        >
          Перейти в корзину
        </a>
        <a
          className="button button--black-border button--small modal__button modal__button--right"
          href="/"
          onClick={[Function]}
        >
          Продолжить покупки
        </a>
      </div>
      <button
        aria-label="Закрыть"
        className="modal__close-btn button-cross"
        data-test="test-close-modal"
        onClick={[MockFunction]}
        type="button"
      >
        <span
          className="button-cross__icon"
        />
        <span
          className="modal__close-btn-interactive-area"
        />
      </button>
    </div>
  </div>
</div>
`;
